packages/bin-etl/src/index.ts

```diff
@@ -0,0 +1,42 @@
+import fs from 'fs'
+
+const fileName = './worlds.json'
+processFile(fileName)
+
+function processFile(n: string) {
+    fs.readFile(n, 'utf8', function (err,data) {
+        if (err) {
+          return console.log(err);
+        }
+        handleJson(JSON.parse(data))
+      });
+}
+
+function handleJson(data: any) {
+    const cluster = data.world.clusters.cluster
+    const selectedFields = cluster.map(buildSelectedField)
+    const stringData = JSON.stringify(selectedFields.filter(isValidZone))
+    fs.writeFileSync("data-dump.json", stringData)
+}
+
+function buildSelectedField(m: any): any {
+  return {
+    "name": m["@displayname"],
+    "type": m["@type"],
+    "resources": extractResources(m),
+    "markers": extractMapMarkers(m),
+  }
+}
+
+function isValidZone(sf: any) {
+  const type: String = sf.type
+  return (type.startsWith("TUNNEL") || type.startsWith("OPENPVP") || type.startsWith("SAFEAREA"))
```

Comment:
I think we want cities and things like that as well. I don't think this accomplishes that. 

Looks like all the important ones start with `PLAYERCITY`. 


![Screen Shot 2020-11-28 at 2 43 47 PM](https://user-images.githubusercontent.com/2829924/100524683-2621e180-3188-11eb-8217-1df0ae1b50fb.png)

Reply:
Re: Discord conversation. I'm ok with keeping the same data structure, but we should still include the cities like above @mmcken3 .